scripts/core/ServiceOsmApiDbHootApiDbConflate.sh

```diff
@@ -142,12 +142,12 @@ if [ "$CONFLATE_DATA" == "true" ]; then
   echo ""
   echo "STEP 8a: Conflating the two datasets over the specified AOI with the SQL changeset workflow..."
   echo ""
-  hoot conflate $HOOT_OPTS -D reader.add.source.datetime=false -D reader.preserve.all.tags=true -D hootapi.db.writer.create.user=true -D hootapi.db.writer.overwrite.map=true -D api.db.email=$HOOT_EMAIL -D convert.bounding.box=$AOI -D reader.conflate.use.data.source.ids.1=true -D reader.conflate.use.data.source.ids.2=true -D preserve.unknown1.element.id.when.modifying.features=true -D osm.map.reader.factory.reader=hoot::OsmApiDbAwareHootApiDbReader -D osm.map.writer.factory.writer=hoot::OsmApiDbAwareHootApiDbWriter -D osmapidb.id.aware.url=$OSM_API_DB_URL $OSM_API_DB_URL "$HOOT_DB_URL/5-secondary-complete-$TEST_NAME" "$HOOT_DB_URL/8a-conflated-$TEST_NAME"
+  hoot conflate $HOOT_OPTS -D reader.add.source.datetime=false -D reader.preserve.all.tags=true -D hootapi.db.writer.create.user=true -D hootapi.db.writer.overwrite.map=true -D api.db.email=$HOOT_EMAIL -D convert.bounding.box=$AOI -D reader.conflate.use.data.source.ids.1=true -D reader.conflate.use.data.source.ids.2=true -D osm.map.reader.factory.reader=hoot::OsmApiDbAwareHootApiDbReader -D osm.map.writer.factory.writer=hoot::OsmApiDbAwareHootApiDbWriter -D osmapidb.id.aware.url=$OSM_API_DB_URL $OSM_API_DB_URL "$HOOT_DB_URL/5-secondary-complete-$TEST_NAME" "$HOOT_DB_URL/8a-conflated-$TEST_NAME"
```

Comment:
This option `preserve.unknown1.element.id.when.modifying.features` was removed previously and causes some unit tests to fail.

Reply:
Which test?  I wonder if you had some orphaned test files turned on that weren't part of source control.  I thought disabled every test that calls this script bc its the db holy grail flow we're not using and is just too hard to maintain over time with all the other changes we've been making.

Reply:
but yeah, either way, that options shouldn't be there.

Reply:
No those are the ones.  I just enabled one to see if it also needed `$HOOT_HOME/tmp` and that is why it was failing.  That is when I noticed the previously deleted option that needed to be removed.  It didn't fix the other tests so I left them disabled.

Reply:
Ok, cool.   Eventually I'll pull all of those tests out.  Just wanted to leave them in for a little while longer, as we're still navigating holy grail, in case there was anything left in them useful.